[docs] [develop] Write-heavy workloads clean up (#15816)
yugabyte/yugabyte-db: docs/content/preview/develop/quality-of-service/transaction-priority.md

* Only applies for transactions using [Fail-on-Conflict](../concurrency-control/#fail-on-conflict) concurrency control policy.
* Only conflict resolution is prioritized, not resource consumption as a part.

{{< /note >}}

## Examples

> Create a [YugabyteDB cluster](../../../quick-start/), and open two separate [ysqlsh](../../../admin/ysqlsh/#starting-ysqlsh) connections to it.

Reviewer: Suggestion:

Create a [YugabyteDB universe](../../../quick-start/) and open two separate [ysqlsh](../../../admin/ysqlsh/#starting-ysqlsh) connections to it.

{{< tip title="Tip - Use YugabyteDB Managed" >}}

> You can create a cluster in the free tier of [YugabyteDB Managed](../../../quick-start-yugabytedb-managed/), and open two *cloud shell* connections to it. These cloud shell connections open up in two different browser tabs, which can be used to do the steps below.

Reviewer 2: ```suggestion
You can create a free cluster with [YugabyteDB Managed](../../../quick-start-yugabytedb-managed/), and open two *cloud shell* connections to it. These cloud shell connections open up in two different browser tabs, which you can use to do the steps that follow.
```

### Transaction priority between concurrent operations

> Consider an example scenario of an maintaining a bank account. Create the accounts table, and insert a row into as follows:

Reviewer: Consider an example scenario of maintaining a bank account. Create the accounts table and insert a row into it, as follows:

  ('kevin','checking', 500);
```

> To set a transaction priority for concurrent transactions, perform a deposit and a withdrawal at the same time and set a higher priority to deposit transactions. To simulate this, perform the two operations concurrently - a withdrawal in one session, and a deposit from another. The deposit transaction starts after the withdrawal is initiated, but occurs before the withdrawal is completed from a separate session as demonstrated in the following table:

Reviewer: Suggestion:

To set a transaction priority for concurrent transactions, perform a deposit and a withdrawal at the same time, and set a higher priority to deposit transactions. To simulate this, perform the two operations concurrently - a withdrawal in one session and a deposit in another session. The deposit transaction starts after the withdrawal has been initiated, but occurs before the withdrawal is completed from a separate session, as demonstrated in the following table:

    (1 row)
    ```

>    As seen from the resulting _High priority transaction_ value in the preceding example, the transaction priority is randomly chosen between the lower and upper bound.

Reviewer 2: It looks like the second `set yb_transaction_priority_lower_bound` line should be setting the upper bound, too, not the lower bound a second time.

```suggestion
   The transaction priority is randomly chosen between the lower and upper bound.
```

    Highest priority transaction
    (1 row)
    ```

> ## Learn more

Reviewer: "Learn more", "Next steps", etc, should never be a header. Instead, there should be a "For more info" paragraph, typically under the top section of the page.

For more information, see the following:

- link 1
- link 2
- link 3
